api/src/main/java/uk/gov/dhsc/htbhf/dwp/service/EligibilityService.java

```diff
@@ -27,17 +27,20 @@
     private final UCHouseholdRepository ucHouseholdRepository;
     private final LegacyHouseholdRepository legacyHouseholdRepository;
     private final HouseholdVerifier householdVerifier;
+    private final EligibilityResponseFactory eligibilityResponseFactory;
 
     public EligibilityService(@Value("${dwp.base-uri}") String baseUri,
                               RestTemplate restTemplate,
                               UCHouseholdRepository ucHouseholdRepository,
                               LegacyHouseholdRepository legacyHouseholdRepository,
-                              HouseholdVerifier householdVerifier) {
+                              HouseholdVerifier householdVerifier,
+                              EligibilityResponseFactory eligibilityResponseFactory) {
```

Comment:
Some refactoring is required here as we now have a constructor with 7 parameters. Didn't want to do it in this PR in order to keep it small.

Reply:
If you don't make `EligibilityResponseFactory` a component, then you don't need to add it as a constructor param and can call its methods statically.

Reply:
Managed to reduce this by creating a DwpClient that encapsulates the rest template and the uri variable.